index.js

```diff
@@ -19,6 +19,7 @@ const SeverityAnnotationLevelMap = new Map([
     const projectFileName = core.getInput("project");
     const pattern = core.getInput("pattern");
     const ghToken = core.getInput("token");
+    const onlyChanged = core.getInput("only-changed") || false;
```

Comment:
This file will be automatically updated. You don't have to change this file

Reply:
I know - but without compiled file - I cannot test this branch in my project